internal/store/endpoint_test.go

```diff
@@ -51,6 +53,10 @@ func TestEndpointStore(t *testing.T) {
 					Labels: map[string]string{
 						"app": "foobar",
 					},
+					Annotations: map[string]string{
+						"whitelisted":     "true",
```

Comment:
The usage of whitelist and blacklist has some rather problematic undertones. As we are trying to be rid of them for the next major release, we should stick to `allowed/denied`.

Reply:
Agreed!